### PR TITLE
Meta index/parts/schema log and comments

### PR DESCRIPTION
**Pull request by pengweisong**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### Description:
Add logs and comments in meta parts/schema/index source files.

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> NA

### CODE REVIEW

File: src/meta/processors/index/FTIndexProcessor.cpp [hunk 1]
```diff
@@ -95,15 +95,20 @@ void CreateFTIndexProcessor::process(const cpp2::CreateFTIndexReq& req) {
     auto indexName = MetaKeyUtils::parsefulltextIndexName(it->key());
     auto indexItem = MetaKeyUtils::parsefulltextIndex(it->val());
     if (indexName == name) {
-      LOG(ERROR) << "Fulltext index exist : " << indexName;
+      LOG(INFO) << "Fulltext index exist : " << indexName;
       handleErrorCode(nebula::cpp2::ErrorCode::E_EXISTED);
       onFinished();
       return;
     }
+<<<<<<< HEAD
```
Comment (liwenhui-soul) [inline src/meta/processors/index/FTIndexProcessor.cpp]: solve the conflict?

**Comment by critical27** [comment]
Compile seems failed~

**Comment by pengweisong** [comment]
Have fixed the compile.

**Review by critical27 — DISMISSED** [review]
Good job~

File: src/meta/processors/parts/CreateSpaceProcessor.cpp [hunk 2]
```diff
@@ -266,7 +266,7 @@ void CreateSpaceProcessor::process(const cpp2::CreateSpaceReq& req) {
   }
```
Comment (liwenhui-soul) [inline src/meta/processors/parts/CreateSpaceProcessor.cpp:267]: CreateSpaceProcessor.cpp:264  should be Vlog, or it's too redundant

File: src/meta/processors/parts/GetPartsAllocProcessor.cpp [hunk 3]
```diff
@@ -77,7 +77,7 @@ std::unordered_map<PartitionID, TermID> GetPartsAllocProcessor::getTerm(GraphSpa
     if (statusVec[i].ok()) {
       std::tie(std::ignore, term, code) = MetaKeyUtils::parseLeaderValV3(vals[i]);
       if (code != nebula::cpp2::ErrorCode::SUCCEEDED) {
-        LOG(WARNING) << apache::thrift::util::enumNameSafe(code);
+        LOG(INFO) << apache::thrift::util::enumNameSafe(code);
         LOG(INFO) << folly::sformat("term of part {} is invalid", partIdVec[i]);
         continue;
       }
```
Comment (liwenhui-soul) [inline src/meta/processors/parts/GetPartsAllocProcessor.cpp:83]: GetPartsAllocProcessor.cpp:84 also need to be Vlog

**Review by liwenhui-soul — APPROVED** [review]

**Review by critical27 — APPROVED** [review]

**Review by liwenhui-soul — APPROVED** [review]

**Pull request merged**